proxy/envoy/testdata/cds-ssl-context.json.golden

```diff
@@ -9,8 +9,7 @@
     "ssl_context": {
      "cert_chain_file": "/etc/certs/cert-chain.pem",
      "private_key_file": "/etc/certs/key.pem",
-     "ca_cert_file": "/etc/certs/root-cert.pem",
-     "verify_subject_alt_name": []
```

Comment:
Yes, you are right, Kuat.
My question is, is this cluster supposed to point to egress? If so, the egress should use Istio auth when talking to internal proxes, right? Then verify_subject_alt_name is required.